mne/tests/test_annotations.py

```diff
@@ -1361,5 +1361,5 @@ def test_annotation_rename():
         raw.rename_annotations({"aaa": "does not exist"})
     with pytest.raises(ValueError, match="[' a']"):
         raw.rename_annotations({" a": "does not exist"})
-    with pytest.raises(ValueError, match="<class 'str'> was provided"):
+    with pytest.raises(TypeError, match="dict, got <class 'str'> instead"):
         raw.rename_annotations("wrong")
```

Comment:
It won't increase coverage to test these twice (once as `annot.rename()` and once as `raw.rename_annotations()`) since the latter is just an alias; it should suffice to call the alias *once*.  But I guess that no longer matters since there seems to be consensus to ditch the alias, so these will get removed anyway.